grafana-plugin/src/containers/RotationForm/RotationForm.tsx

```diff
@@ -422,7 +424,7 @@ export const RotationForm = observer((props: RotationFormProps) => {
       setIsLimitShiftEnabled(value);
 
       if (!value) {
-        if (isMaskedByWeekdays && shiftEnd.diff(shiftStart, 'hours') > 24) {
+        if (isMaskedByWeekdays) {
```

Comment:
![image](https://github.com/grafana/oncall/assets/20116910/10f2964d-10e6-4db9-9526-651058dba8da)


When `Limit each shift length` is disabled and `Mask by weekdays` is enabled, `shift_end` should always be `shift_start + 24 hours`.

Reply:
I remember this was needed, was added by Maxim originally as a fix for another escalation

Reply:
AFAICT [the same check in another place](https://github.com/grafana/oncall/blob/3f294c266cd4c979e030076e1d59ebb02e9950ff/grafana-plugin/src/containers/RotationForm/RotationForm.tsx#L398) makes sense, but the one I changed in the PR is not needed there

Reply:
Got it, all good 👍